intelsdi-x/snap-plugin-collector-ceph
# Update metrics list to Ceph Jewel (generated on 10.2.3)

Fixes #16 

Summary of changes: **not backwards compatible**
- Update `metrics.go` for with Jewel `perf schema` definition
- Update documentation with updated `perf schema` definition
- Update example task file with new metric names

How to verify it:
- Run against a Jewel cluster and check that metrics are being fetched

Testing done:
- Ran against a Ceph running Jewel (10.2.3) and made sure metrics are reported correctly

**Note** There might have been ways of keeping backwards compatibility, but after chatting in the Slack channel, it was suggested to drop backwards compatibility.
